db/migrate/20190715235659_create_comments.rb

```diff
@@ -0,0 +1,11 @@
+class CreateComments < ActiveRecord::Migration[5.2]
+  def change
+    create_table :comments do |t|
+      t.text :content
+      t.integer :user_id
```

Comment:
This should be the same as you add user to post. 👇 
```ruby
class AddUserToPost < ActiveRecord::Migration[5.2]
  def change
    add_reference :posts, :user, foreign_key: true
   end
end
```
The way you added user_id and post_id is not a reference. So you have to do the same things as you did for post migration that code I mentioned above.